### PR TITLE
Updated the broken links in TF doc

### PR DESCRIPTION
**Pull request by sushreebarsa**
Changed the broken links with correct links for the following texts in making_new_layers_and_models_via_subclassing.py file in documentation. 
1.guide to writing a training loop from scratch, 
2. Functional model 
3.guide to saving and serializing models
4.understanding padding and masking

Thank you!

### CODE REVIEW

**Review by MarkDaoust — CHANGES_REQUESTED** [review]
Thanks for pointing this out. 

The files in the first level of /guides/ are mirrored to tensorflow.org.

This will be best is I just fix the import tool, since there are similar errors in a bunch of other files. 

We should be consistent here.  Either all links should or the `(/guides/{name}/)` form (no change), or you could update them all to be keras.io URLs, so they also work in colab.

I'll fix tensorflow.org

**Comment by MarkDaoust** [comment]
cl/549120156

**Comment by MarkDaoust** [comment]
The links on tensorflow.org are fixed, let's close this.

**Comment by sushreebarsa** [comment]
@MarkDaoust Thank you for the update!
